go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/Azure/kubelogin
 
 // NOTE: kubelogin follows the same support policy as Go, which supports the last two major versions.
-go 1.21.11
+go 1.23.4
```

Comment:
this is unexpected? Or we should bump this to 1.22.x to match the support policy comment above